superbench/runner/runner.py

```diff
@@ -387,6 +396,30 @@ def __merge_monitor_metrics(self, node_path):
 
         return metrics_summary
 
+    def _generate_ibstat(self):
```

Comment:
Why not move this function to gen_traffic_pattern_config.py?

Reply:
addressed